src/lib/components/Time.svelte

```diff
@@ -215,11 +216,12 @@
    * @param {any} e
    */
   function onClick(e) {
+    console.log('on click', minuteIncrement)
     if (!canSelect || !e.target) return;
     if ((e.type === 'mousemove' && !handleMoveMove) || (!isMinuteView && e.target.tagName !== 'BUTTON')) return;
     let a = 0;
     let b = 0;
-    if (e.target.tagName === 'BUTTON') {
+    if (e.target.tagName === 'BUTTON' && minuteIncrement === 1) {

```

Comment:
Also this condition is not good, because when you change the `minuteIncrement` value and want to change hour, it's not possible.